src/main/java/hillel/dzhurynskyi/hm36/controller/OrderController.java

```diff
@@ -0,0 +1,34 @@
+package hillel.dzhurynskyi.hm36.controller;
+
+import hillel.dzhurynskyi.hm36.entity.Order;
+import hillel.dzhurynskyi.hm36.repository.OrderRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
```

Comment:
can be @Controller too to keep style consistent

Reply:
> can be @controller too to keep style consistent
@mkharkhuta it not working if we anotate like @controller, it return errors